5-WebApp-AuthZ/5-2-Groups/Controllers/UserProfileController.cs

```diff
@@ -8,6 +9,7 @@
 
 namespace WebApp_OpenIDConnect_DotNet.Controllers
 {
+    // [Authorize(Roles = "8873daa2-17af-4e72-973e-930c94ef7549")] // Using groups ids in the Authorize attribute
```

Comment:
It may not be related to this line, but I got this error when I tried to access UserProfileController:
```
An unhandled exception occurred while processing the request.
ServiceException: Code: NoPermissionsInAccessToken
Message: The token contains no permissions, or permissions can not be understood.
```